core/fee/engine_test.go

```diff
@@ -77,7 +76,6 @@ func TestFeeEngine(t *testing.T) {
 	t.Run("calculate batch auction trading fee empty trade", testCalcBatchAuctionTradingErrorEmptyTrade)
 	t.Run("calculate batch auction trading fee same batch", testCalcBatchAuctionTradingSameBatch)
 	t.Run("calculate batch auction trading fee different batches", testCalcBatchAuctionTradingDifferentBatches)
-	t.Run("calculate position resolution", testCalcPositionResolution)
```

Comment:
was there no chances to port that tests / write a new one instead of remove it completely?

Reply:
Not really. The old fee calculation calculated the amounts per distressed party (pro-rated WRT how much of the fees they actually have to end up paying). After checking with research, we're now paying the whole fee from the insurance pool, so close to none of the logic/calculations we were testing here remain

Reply:
Doesn't mean it's not worth a test for the whole bunch of logic added / moved around.

Reply:
Tests are already in the pipeline on next PR branch